### PR TITLE
[INFRA] pass benchmark_min_time to performance tests

### PR DESCRIPTION
**Pull request by eseiler**
This should decrease the time needed for performance CI. (16m -> 2m)

I added an option to enable/disable this.

### CODE REVIEW

File: test/performance/CMakeLists.txt [hunk 1]
```diff
@@ -10,14 +10,20 @@ project (seqan3_test_performance CXX)
 
 include (../seqan3-test.cmake)
 
+option (SEQAN3_FAST_TESTS "Run each benchmark for a shorter time. Timings are unreliable." OFF)
```
Comment (marehr) [inline test/performance/CMakeLists.txt:13]:
```suggestion
option (SEQAN3_BENCHMARK_MIN_TIME "Set --benchmark_min_time= for each bechmark. Timings are unreliable in CI." 1)
```

File: test/performance/CMakeLists.txt [hunk 2]
```diff
@@ -10,14 +10,20 @@ project (seqan3_test_performance CXX)
...
     seqan3_test_component (test_name "${benchmark}" TEST_NAME)
 
     add_executable (${target} ${benchmark_cpp})
     target_link_libraries (${target} seqan3::test::performance)
-    add_test (NAME "${test_name}" COMMAND ${target})
+    if (SEQAN3_FAST_TESTS)
+        add_test (NAME "${test_name}" COMMAND ${target} "--benchmark_min_time=0.01")
+    else ()
+        add_test (NAME "${test_name}" COMMAND ${target})
+    endif ()
```
Comment (marehr) [inline test/performance/CMakeLists.txt:26]:
```suggestion
    add_test (NAME "${test_name}" COMMAND ${target} "--benchmark_min_time=${SEQAN3_BENCHMARK_MIN_TIME}")
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -263,7 +263,7 @@ jobs:
         run: |
           mkdir seqan3-build
           cd seqan3-build
-          cmake ../seqan3/test/${{ matrix.build }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_FLAGS="${{ matrix.cxx_flags }}" -DSEQAN3_VERBOSE_TESTS=OFF
+          cmake ../seqan3/test/${{ matrix.build }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_FLAGS="${{ matrix.cxx_flags }}" -DSEQAN3_VERBOSE_TESTS=OFF -DSEQAN3_FAST_TESTS=ON
```
Comment (marehr) [inline .github/workflows/ci.yml:266]:
```suggestion
          cmake ../seqan3/test/${{ matrix.build }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_FLAGS="${{ matrix.cxx_flags }}" -DSEQAN3_VERBOSE_TESTS=OFF -DSEQAN3_BENCHMARK_MIN_TIME=0.001
```

**Review by marehr — CHANGES_REQUESTED** [review]
I would simply pass this parameter instead of making it a boolean option.

File: .travis.yml [hunk 4]
```diff
@@ -78,7 +78,7 @@ install:
 before_script:
   - mkdir ../seqan3-build
   - cd ../seqan3-build
-  - cmake ../seqan3/test/${BUILD} -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DCMAKE_CXX_FLAGS="-fcompare-debug-second ${CXXFLAGS}" -DSEQAN3_VERBOSE_TESTS=OFF
+  - cmake ../seqan3/test/${BUILD} -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DCMAKE_CXX_FLAGS="-fcompare-debug-second ${CXXFLAGS}" -DSEQAN3_VERBOSE_TESTS=OFF -DSEQAN3_FAST_TESTS=ON
```
Comment (marehr) [inline .travis.yml]:
```suggestion
  - cmake ../seqan3/test/${BUILD} -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DCMAKE_CXX_FLAGS="-fcompare-debug-second ${CXXFLAGS}" -DSEQAN3_VERBOSE_TESTS=OFF -DSEQAN3_BENCHMARK_MIN_TIME=0.001```

**Review by marehr — APPROVED** [review]

**Review by smehringer — APPROVED** [review]

**Pull request merged**